src/tmt_web/generators/templates/_base.html.j2

```diff
@@ -0,0 +1,10 @@
+<!-- _base.html.j2 -->
```

Comment:
Shouldn't the doctype be defined on the first line?

```
<!DOCTYPE html>
```

Reply:
Thanks for reviewing this! Could you fix the jinja-related things in a commit? :)